package.json

```diff
@@ -9,7 +9,6 @@
     "url": "https://github.com/skatejs/skatejs"
   },
   "main": "umd/skatejs.js",
-  "module": "es/index.js",
```

Comment:
so with this I am unable to tree shake skate right? I would have to explicitly alias it in webpack for instance ( with which I'd have to use native-shim though )